src/components/uielements/label/label.stories.tsx

```diff
@@ -0,0 +1,182 @@
+        </Label>
+        <Label size="normal" color="normal" weight="light">
+          This is the LABEL!
+        </Label>
+        <Label size="normal" color="normal" weight="bold">
+          This is the LABEL!
+        </Label>
+        <Label size="normal" color="primary">
+          This is the LABEL!
+        </Label>
+        <Label size="normal" color="success">
+          This is the LABEL!
+        </Label>
+        <Label size="normal" color="warning">
+          This is the LABEL!
+        </Label>
+        <Label size="normal" color="error">
+          This is the LABEL!
+        </Label>
+        <Label size="normal" color="normal">
+          This is the LABEL!
+        </Label>
+        <Label size="normal" color="dark">
+          This is the LABEL!
+        </Label>
+        <Label size="normal" color="light">
+          This is the LABEL!
+        </Label>
+      </div>
+    )
+  })
+  .add('big', () => {
+    return (
+      <div>
+        <Label size="big" color="normal">
+          This is the LABEL!
+        </Label>
+        <Label size="big" color="normal" weight="light">
+          This is the LABEL!
+        </Label>
+        <Label size="big" color="normal" weight="bold">
+          This is the LABEL!
+        </Label>
+        <Label size="big" color="primary">
+          This is the LABEL!
+        </Label>
+        <Label size="big" color="success">
+          This is the LABEL!
+        </Label>
+        <Label size="big" color="warning">
+          This is the LABEL!
+        </Label>
+        <Label size="big" color="error">
+          This is the LABEL!
+        </Label>
+        <Label size="big" color="normal">
+          This is the LABEL!
+        </Label>
+        <Label size="big" color="dark">
+          This is the LABEL!
+        </Label>
+        <Label size="big" color="light">
+          This is the LABEL!
+        </Label>
+      </div>
+    )
+  })
+  .add('large', () => {
+    return (
+      <div>
+        <Label size="large" color="normal">
+          This is the LABEL!
+        </Label>
+        <Label size="large" color="normal" weight="light">
+          This is the LABEL!
+        </Label>
+        <Label size="large" color="normal" weight="bold">
+          This is the LABEL!
+        </Label>
+        <Label size="large" color="primary">
+          This is the LABEL!
+        </Label>
+        <Label size="large" color="success">
+          This is the LABEL!
+        </Label>
+        <Label size="large" color="warning">
+          This is the LABEL!
+        </Label>
+        <Label size="large" color="error">
+          This is the LABEL!
+        </Label>
+        <Label size="large" color="normal">
+          This is the LABEL!
+        </Label>
+        <Label size="large" color="dark">
+          This is the LABEL!
+        </Label>
+        <Label size="large" color="light">
+          This is the LABEL!
+        </Label>
+      </div>
+    )
+  })
+  .add('properties', () => {
```

Comment:
Since you have provided a story by using properties via knobs - which is great -, why do we still need stories for `small`, `normal`, `big`? I would suggest to remove other stories to have `properties` story only. Just keep it simple!

Reply:
Okay, I removed other stories